Service/Client.php

```diff
@@ -37,13 +37,63 @@ public function __construct(Elephant $elephantIO)
     }
 
     /**
-    * Send to socketio
-    * @param string $eventName event name
-    * @param mixed  $data      data to send must be serializable
-    */
-    public function send($eventName, $data)
+     * Initialize client
+     *
+     * @return self
+     */
+    public function initialize()
+    {
+        $this->elephantIO->initialize();
+
+        return $this;
+    }
+
+    /**
+     * Set namespace for event
+     *
+     * @param $namespace
+     *
+     * @return self
+     */
+    public function of($namespace)
+    {
+        $this->elephantIO->of($namespace);
+
+        return $this;
+    }
+
+    /**
+     * Emit message to socket.io
+     *
+     * @param string $event
+     * @param array  $args
+     * @return $this
+     */
+    public function emit($event, array $args)
+    {
+        $this->elephantIO->emit($event, $args);
+
+        return $this;
+    }
+
+    /**
+     * Close client
+     */
+    public function close()
+    {
+        $this->elephantIO->close();
+    }
+
+    /**
+     * Send to socket.io
+     * @param string $eventName event name
+     * @param mixed  $data      data to send must be serializable
+     * @param string $namespace namespace for event
+     */
+    public function send($eventName, $data, $namespace = '/')
```

Comment:
could you set a default vault to null for namespace and use an if condition after